backend/server.py

```diff
@@ -1,18 +1,94 @@
-from flask import Blueprint, request, jsonify
+from flask import Flask, Blueprint, request, jsonify, session
 from flask_cors import CORS
-from app.models.predict import predict_disease
+
+from flask_bcrypt import Bcrypt
 from flask_sqlalchemy import SQLAlchemy
-from datetime import datetime
 
-bp = Blueprint('main', __name__)
+import os
+
+from app.models.users.user import db, User
+from app.routes import register_all_blueprints
+from app.models.predict import predict_disease
+
+app = Flask(__name__)
+
+import secrets
+sk = secrets.token_hex(16)
+app.config['SECRET_KEY'] = sk
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql://root:Password123*@localhost/animalia'
+
+SQLALCHEMY_TRACK_MODIFICATIONS = False 
+SQLALCHEMY_ECHO = True
+
+bcrypt = Bcrypt(app)
+CORS(app, supports_credentials=True)
+
+db.init_app(app)
+
+with app.app_context():
+    db.create_all()
+
+@app.route('/users', methods=['GET'])
+def listuser():
+    all_users = User.query.all()
+    results = [{"id": user.id, "name": user.name, "email": user.email, "Created At": user.date_added} for user in all_users]
+    return jsonify(results)
```

Comment:
Currently, these APIs are open(not authorized). Are you guys planning to add some kind of token/JWT authentication in it?

Reply:
it was just for testing for now. we will later add authentication